services/httpd/default.nix

```diff
@@ -10,9 +10,7 @@ let
   # Everything else is promoted to HTTPS
   acmeVhost = {
     inherit adminAddr;
-    hostName = tld;
     serverAliases = ["*"];
-    listen = [{ port = 80; }];
```

Comment:
this covered by default or what?

Reply:
if you don't specify any of the SSL options it only listens on port 80. the listen option itself got deprecated.